### PR TITLE
feature/AT-SCE-UI_WebToken

### PR DESCRIPTION
**Pull request by jaimeIxba**
We sent the web token to the USERS & HISTORIES endpoints in the header request.

### CODE REVIEW

**Review by DianaArceoSuarez — APPROVED** [review]

**Review by FlavioAlejandro — APPROVED** [review]

File: src/services/at-sce-api.service.js [hunk 1]
```diff
@@ -29,15 +29,26 @@ const AT_SSO_SERVICE_URI_ENABLED = process.env.AT_SSO_SERVICE_URI_ENABLED;
 const AT_SERVICE_URI = (AT_SSO_SERVICE_URI_ENABLED == 'true') ? AT_SSO_SERVICE_URI : AT_SCE_SERVICE_URI;
 console.log(`at-sce-api.service - AT_SERVICE_URI: ${AT_SERVICE_URI}`);
 
+// AT_SSO_WEB_TOKEN_CLIENT & SECRET
+const AT_SSO_WEB_TOKEN_CLIENT = process.env.AT_SSO_WEB_TOKEN_CLIENT;
+const AT_SSO_WEB_TOKEN_SECRET = process.env.AT_SSO_WEB_TOKEN_SECRET;
+
+const token = `${AT_SSO_WEB_TOKEN_CLIENT}:${AT_SSO_WEB_TOKEN_SECRET}`;
+const access_token = Buffer.from(token).toString('base64');
```
Comment (at-internship) [inline src/services/at-sce-api.service.js]: `const authorization_token = Buffer.from(token).toString('base64');`

File: src/services/at-sce-api.service.js [hunk 2]
```diff
@@ -29,15 +29,26 @@ const AT_SSO_SERVICE_URI_ENABLED = process.env.AT_SSO_SERVICE_URI_ENABLED;
 const AT_SERVICE_URI = (AT_SSO_SERVICE_URI_ENABLED == 'true') ? AT_SSO_SERVICE_URI : AT_SCE_SERVICE_URI;
 console.log(`at-sce-api.service - AT_SERVICE_URI: ${AT_SERVICE_URI}`);
 
+// AT_SSO_WEB_TOKEN_CLIENT & SECRET
+const AT_SSO_WEB_TOKEN_CLIENT = process.env.AT_SSO_WEB_TOKEN_CLIENT;
+const AT_SSO_WEB_TOKEN_SECRET = process.env.AT_SSO_WEB_TOKEN_SECRET;
+
+const token = `${AT_SSO_WEB_TOKEN_CLIENT}:${AT_SSO_WEB_TOKEN_SECRET}`;
+const access_token = Buffer.from(token).toString('base64'); 
+console.debug("encodedToken-->", access_token);
```
Comment (at-internship) [inline src/services/at-sce-api.service.js]: `console.debug("authorization_token-->", authorization_token);`

File: src/services/at-sce-api.service.js [hunk 3]
```diff
@@ -29,15 +29,26 @@ const AT_SSO_SERVICE_URI_ENABLED = process.env.AT_SSO_SERVICE_URI_ENABLED;
...
+  var qs = require('qs');
+  
   return axios({
     method: "POST",
     url: `${AT_SERVICE_URI}/v1/login`,
-    data: data,
     headers: {
-      "content-type": "application/json",
+      'Content-Type': 'application/x-www-form-urlencoded',
+      'Authorization': `Basic ${access_token}`
```
Comment (at-internship) [inline src/services/at-sce-api.service.js]: `Authorization': `Basic ${authorization_token}`

File: src/services/at-sce-api.service.js [hunk 4]
```diff
@@ -47,7 +58,8 @@ AT_SCE_SERVICE.getAllUsers = () => {
     method: "GET",
     url: `${AT_SERVICE_URI}/v1/users`,
     headers: {
-      "content-type": "application/json",
+      "Content-Type": "application/json",
+      'Authorization': `Basic ${access_token}`
```
Comment (at-internship) [inline src/services/at-sce-api.service.js]: `Authorization': `Basic ${authorization_token}`

File: src/services/at-sce-api.service.js [hunk 5]
```diff
@@ -58,7 +70,8 @@ AT_SCE_SERVICE.getUserById = (id) => {
     method: "GET",
     url: `${AT_SERVICE_URI}/v1/users/${id}`,
     headers: {
-      "content-type": "application/json",
+      "Content-Type": "application/json",
+      'Authorization': `Basic ${access_token}`
```
Comment (at-internship) [inline src/services/at-sce-api.service.js]: `Authorization': `Basic ${authorization_token}`

File: src/services/at-sce-api.service.js [hunk 6]
```diff
@@ -70,7 +83,8 @@ AT_SCE_SERVICE.createUser = (data) => {
     url: `${AT_SERVICE_URI}/v1/users`,
     data: data,
     headers: {
-      "content-type": "application/json",
+      "Content-Type": "application/json",
+      'Authorization': `Basic ${access_token}`
```
Comment (at-internship) [inline src/services/at-sce-api.service.js]: `Authorization': `Basic ${authorization_token}`

File: src/services/at-sce-api.service.js [hunk 7]
```diff
@@ -82,7 +96,8 @@ AT_SCE_SERVICE.updateUser = (data) => {
     url: `${AT_SERVICE_URI}/v1/users/${data.id}`,
     data: data,
     headers: {
-      "content-type": "application/json",
+      "Content-Type": "application/json",
+      'Authorization': `Basic ${access_token}`
```
Comment (at-internship) [inline src/services/at-sce-api.service.js]: `Authorization': `Basic ${authorization_token}`

File: src/services/at-sce-api.service.js [hunk 8]
```diff
@@ -93,30 +108,37 @@ AT_SCE_SERVICE.deleteUser = (id) => {
     method: "DELETE",
     url: `${AT_SERVICE_URI}/v1/users/${id}`,
     headers: {
-      "content-type": "application/json",
+      "Content-Type": "application/json",
+      'Authorization': `Basic ${access_token}`
```
Comment (at-internship) [inline src/services/at-sce-api.service.js]: `Authorization': `Basic ${authorization_token}`

**Review by at-internship — CHANGES_REQUESTED** [review]
Please change `Authorization': `Basic ${authorization_token}`

**Review by at-internship — APPROVED** [review]

**Pull request merged**